.yarn/patches/@metamask-network-controller-npm-20.2.0-98b1a5ae59.patch

```diff
@@ -0,0 +1,33 @@
+diff --git a/PATCH.txt b/PATCH.txt
+new file mode 100644
+index 0000000000000000000000000000000000000000..78b9156dc2b0bf7c33dadf325cb3ec0bfae71ccb
+--- /dev/null
++++ b/PATCH.txt
+@@ -0,0 +1,3 @@
++We remove `lookupNetwork` from `initializeProvider` in the network controller to prevent network requests before user onboarding is completed.
++The network lookup is done after onboarding is completed, and when the extension reloads if onboarding has been completed.
++This patch is part of a temporary fix that will be reverted soon to make way for a more permanent solution. https://github.com/MetaMask/metamask-extension/pull/23005
```

Comment:
I don't know the context of this patch, but it was here before so I'm maintaining it.